src/doc/unstable-book/src/compiler-flags/env.md

```diff
@@ -20,6 +21,21 @@ Then you will have:
 assert_eq!(env!("PATH"), "env");
 ```
 
+It will trigger a new compilation if any of the `--env` argument value is different.
+So if you first passed:
```

Comment:
I think it should mention that it will end up in depinfo, which is the interesting part. The fact that rustc will also recompile it itself should be obvious and doesn't need an example.

Reply:
Doesn't hurt to have an example I suppose? :)

Adding the dep info.